<launch>
	<rosparam file="/home/thanh-long-vu/github/PhD/param/ROS_TCP_Connector_Params.yaml" command="load"/>

    <node pkg="robotics_demo" name="server_endpoint" type="server_endpoint.py"/>

    <include file="/home/thanh-long-vu/github/PhD/src/realsense-ros/realsense2_camera/launch/rs_camera.launch"/>

    <node pkg="image_transport" name="color_image_compressor" type="republish" args="raw in:=/camera/color/image_raw compressed out:=/compressed/color">
    </node>

    <node pkg="image_transport" name="deth_image_compressor" type="republish" args="raw in:=/camera/depth/image_rect_raw compressed out:=/compressed/depth">
    </node>
</launch>